warehouse/models/mart/gtfs/_mart_gtfs_fcts.yml

```diff
@@ -263,9 +263,10 @@ models:
       - name: vehicle_license_plate
         description: |
           See: https://gtfs.org/realtime/reference/#message-vehicledescriptor.
-      - name: vehicle_wheelchair_accessible
-        description: |
-          See: https://gtfs.org/realtime/reference/#message-vehicledescriptor.
+        # Should this exist? It does not currently in the model.
```

Comment:
Yes, can you write a ticket to add it? This is part of the spec and given our interest in accessibility information we should have it. 
![image](https://user-images.githubusercontent.com/55149902/220937741-7475df3f-090c-468f-a576-f7273dc3e891.png)

Reply:
I'll just do it here, it's just not in the mart.